config/base.yaml

```diff
@@ -12,3 +12,4 @@ email:
   password_reset_email_subj: "Reset password"
 app_url: "auth.unicsmcr.com"
 data_policy_url: "https://drive.google.com/file/d/1wMcJbfEhIp9FjdNbyom4RVUoTH4xc0OB/view"
+MaxTeamMembers: "Soft Maximum Team Members"
```

Comment:
The idea behind this setting is that we would store the number of people in a team so it would need to be an int rather than a string and by default the value would be 4.

Reply:
Actually, make it a `uint` to prevent people from entering negative values.